punchpipe/control/scheduler.py

```diff
@@ -11,6 +12,7 @@ def generic_scheduler_flow_logic(
     query_ready_files_func, construct_child_file_info, construct_child_flow_info, pipeline_config_path,
         update_input_file_state=True, new_input_file_state="progressed",
         session=None, reference_time: datetime | None = None,
+        children_are_one_to_one: bool = False,
```

Comment:
Please document this `children_are_one_to_one` option. We should have a better docstring for this function in general. (I should've done that when I wrote this function.)